Monika After Story/game/script-ch30.rpy

```diff
@@ -977,7 +975,9 @@ label spaceroom(start_bg=None, hide_mask=None, hide_monika=False, dissolve_all=F
     # TODO: move this to bday autoload
     if persistent._mas_player_bday_decor:
         $ store.mas_surpriseBdayShowVisuals()
-    else:
+
+    # both 922 and pbday share the same visual funcs, so need to check both before hiding
+    if not persistent._mas_bday_visuals and not persistent._mas_player_bday_decor:
```

Comment:
just leaving a note here that the mas_bday_visuals flag will be the sole flag used to toggle visuals in the future - the bday_decor flag is supposed to be moved at some point.